examples/01_plotting/plot_surf_stat_map.py

```diff
@@ -0,0 +1,122 @@
+"""
+Demonstrate basic loading and plotting of cortical surface data
```

Comment:
I think that the title of this example could be improved: the example isn't that basic: it included array manipulation to build a seed-based regression.

Maybe I would suggest "Seed-based connectivity on the surface"